addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/Manager.java

```diff
@@ -661,7 +665,8 @@ public synchronized void foundDir(
                 }
             }
 
-            System.out.println("Dir found: " + url.getFile() + " - " + statusCode);
+            LOG.info("Dir found: " + url.getFile() + " - " + statusCode);
```

Comment:
Should be debug, otherwise it's too verbose by default. DirBuster doesn't differentiate from dirs/files found while spidering from dirs/files found with the provided list so logging everything as info is not that useful.

Reply:
Corrected. Will be in the next commit.